helpers/migrate.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Since we've replaced this script with https://github.com/terraform-google-modules/terraform-google-project-factory/pull/35 can we delete this file?